design/work_item_children.go

```diff
@@ -0,0 +1,23 @@
+package design
+
+import (
+	d "github.com/goadesign/goa/design"
+	a "github.com/goadesign/goa/design/apidsl"
+)
+
+var _ = a.Resource("work-item-children", func() {
```

Comment:
But @aslakknutsen and I recommended to put the list-children action in the work item resource. You've placed it in the `WorkItemLinks` resource. Why?